src/applications/personalization/profile-2/components/connected-apps/ConnectedAppDeleteModal.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+
+export function ConnectedAppDeleteModal({
+  appName,
+  deleting,
+  modalOpen,
+  onCloseModal,
+  onConfirmDelete,
+}) {
+  return (
+    <Modal
+      clickToClose
+      cssClass="va-modal"
```

Comment:
I'll make sure to tinker with this modal a bit more tomorrow to match the designs